src/test/resources/com/cloudbees/jenkins/plugins/bitbucket/client/events/BitbucketCloudPullRequestEventTest/rejectedPayload.json

```
@@ -90,7 +90,7 @@
         "hash": "63e3d18dca4c",
         "links": {
           "self": {
-            "href": "https://api.bitbucket.org/2.0/repositories/stephenc/temp-fork/commit/63e3d18dca4c"
+            "href": "https://api.bitbucket.org2.0/repositories/%7Bf70f195e-ade1-4961-9f89-547541377b80%7D/temp-fork/commit/63e3d18dca4c"
```

Looks like a typo that was copied to many places
```suggestion
            "href": "https://api.bitbucket.org/2.0/repositories/%7Bf70f195e-ade1-4961-9f89-547541377b80%7D/temp-fork/commit/63e3d18dca4c"
```

---

eek..."over 101+ occurances"

---

search and replace? :laughing: